src/modules/trading/components/trading--form/trading--form.styles.less

```diff
@@ -63,31 +77,47 @@
 }
 
 .TradingForm__error_message_container {
-  background: rgba(255, 39, 39, 0.2);
-  border: 1px solid rgba(255, 39, 39, 0.2);
+  background: @color-error-message-bright-fade;
+  border: 1px solid @color-error-text;
   border-radius: 3px;
   box-sizing: border-box;
   display: block;
   font-size: @font-size-small;
   line-height: @font-size-medium;
   margin: 1rem;
-  padding: 0.5rem;
+  padding: 0.75rem;
 }
 
 .TradingForm__error_message {
-  color: @color-error-text;
+  .text-12-bold;
+
+  color: @color-secondary-text;
   margin-bottom: 0.25rem;
 
   &:last-of-type {
     margin-bottom: 0;
   }
 
-  svg {
+  > svg {
     margin: 0 0.2rem;
-    min-height: 1.75em;
-    min-width: 1.75em;
-    vertical-align: middle;
+    min-height: 1rem;
+    min-width: 1rem;
+    vertical-align: text-bottom;
```

Comment:
<img width="318" alt="Screen Shot 2019-03-19 at 7 26 15 PM" src="https://user-images.githubusercontent.com/6775839/54654880-f8147080-4a7c-11e9-8dce-c4a4ef34c09f.png">
i don't think this vertical alignment is working here. the text and svg are centered together. this is how it looks in the mock but probably figma being weird?

Reply:
error in UI looks fine to me. in the mock it's not quite lined up correctly. It's not bottom aligned and not center aligned. In UI I went with text-bottom aligned.